packages/config/src/projects/jetstreamchain/jetstreamchain.ts

```diff
@@ -0,0 +1,32 @@
+import { ProjectId, UnixTime } from '@l2beat/shared-pure'
+import type { ScalingProject } from '../../internalTypes'
+import { underReviewL3 } from '../../templates/underReview'
+
+export const jetstreamchain: ScalingProject = underReviewL3({
+  id: 'jetstreamchain',
+  capability: 'universal',
+  addedAt: UnixTime(1753947052),
+  hostChain: ProjectId('arbitrum'),
+  display: {
+    name: 'Jetstream Chain',
```

Comment:
add a `shortName` with just "Jetstream" given that this name is already too long